src/gameplay/gameTimer.ts

```diff
@@ -136,6 +138,10 @@ export class GameTimer {
     return this.dateTimerExpires;
   }
 
+  isTimerSet() {
```

Comment:
Maybe actually rename this to `isTimerActive()`. Set is a big ambiguous here because in this case we can have a valid timer set, but it's not active, e.g. in a voided game or in a paused game. 

Timer Active serves what we want I believe and is a lot clearer.